src/test/java/bio/terra/service/iam/IamServiceTest.java

```diff
@@ -35,32 +38,63 @@ public void setup() {
         iamService = new IamService(iamProvider, configurationService);
     }
 
-
     @Test
-    public void testAddPolicyMember() throws InterruptedException{
+    public void testAddPolicyMember() throws InterruptedException {
+        var authenticatedUserRequest = new AuthenticatedUserRequest();
         var policyModel = new PolicyModel();
-        when(iamProvider.addPolicyMember(any(), any(), any(), any(), any())).thenReturn(policyModel);
+        String policyName = "policyName";
+        String email = "email";
+        when(iamProvider.addPolicyMember(
+                eq(authenticatedUserRequest),
```

Comment:
It bugs me that we have to provide the same parameter list twice. What if we use `any()` for the `when` and use the actual args for the `verify()`?

Or, in this case the code under test calls `iamProvider.addPolicyMember()` but then never does anything with the result except return it. So this test _could_ skip `when()` and only use `verify()`. It would test the call but not test the `return policy;` line.